test/spec/utils/analyzers/time_tracker_spec.rb

```diff
@@ -191,7 +220,7 @@ def previous_list
     end
 
     def previous_date
-      actions.last.date.to_date
+      actions.last.date.to_datetime.utc.to_date
```

Comment:
I believe this is redundant. Can you confirm that the to_datetime.utc is not required?